torch/csrc/jit/frontend/lexer.h

```diff
@@ -286,6 +286,7 @@ struct TORCH_API SharedParserData {
     switch (kind) {
       case '?':
       case TK_POW:
```

Comment:
should this return True too?

Reply:
Sorry I'm not sure about your question, did you mean `TK_POW`？It do returns true, the switch-case will fall through

My fix is add true case for `TK_IF`, other case is not changed

Reply:
Isn't TK_POW return true in the original statement?

Reply:
TK_POW should be right associative `x**2**3 == x**(2**3)`
it return true in the original statement
it return true in my PR
I didn't changed it